test/Confluent.Kafka.StrongName.UnitTests/Confluent.Kafka.StrongName.UnitTests.csproj

```diff
@@ -25,6 +25,9 @@
 
   <ItemGroup>
     <PackageReference Include="librdkafka.redist" Version="0.11.1" />
+  </ItemGroup>
```

Comment:
Was this intentional? Does it do anything?

Reply:
It doesn't do anything, just a matter of preference. This is actually a reversion back to how it was in 0.11.0. The unit tests cover the change.